CMakeLists.txt

```diff
@@ -91,37 +100,64 @@ else()
     endif()
 endif()
 
+# define sources and most of install targets
+
+## extension modules
+file(GLOB GEOCODERS_SRC geocoders/*.json geocoders/*.md geocoders/*.py)
+file(GLOB GEOCODERS_EXTRA LIST_DIRECTORIES false geocoders/test/*)
+install(FILES ${GEOCODERS_SRC} DESTINATION ${DATADIR}/geocoders)
+
+file(GLOB GUIDES_SRC guides/*.json guides/*.md guides/*.qml guides/*.py)
+file(GLOB GUIDES_EXTRA LIST_DIRECTORIES false guides/test/*)
+install(FILES ${GUIDES_SRC} DESTINATION ${DATADIR}/guides)
+
+file(GLOB MAPS_SRC maps/*.json maps/*.md)
+install(FILES ${MAPS_SRC} DESTINATION ${DATADIR}/maps)
+
+file(GLOB ROUTERS_SRC routers/*.json routers/*.md routers/*.qml routers/*.py)
+file(GLOB ROUTERS_EXTRA LIST_DIRECTORIES false routers/test/*)
+install(FILES ${ROUTERS_SRC} DESTINATION ${DATADIR}/routers)
+
+## python sources: installed in subdir
+file(GLOB POOR_SRC poor/*.py)
+
+## QML sources: installed in subdir
+file(GLOB QML_SRC qml/*.qml)
```

Comment:
Idem, but in `qml/CMakeLists.txt`?

Reply:
See comment at https://github.com/rinigus/pure-maps/pull/531#discussion_r606683158